### PR TITLE
Enha: Read sentry cocoa version at runtime

### PR DESCRIPTION
**Pull request by denrase**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring

## :scroll: Description

- Read version info for cocoa SDK at runtime.
- Update iOS sample project to use current pod.

## :bulb: Motivation and Context

Prior, version info had to be updated in dart code along with updating cocoa sdk.

## :green_heart: How did you test it?

Ran the sample project and verified the correct version was read when recording events.

Bildschirmfoto 2020-11-28 um 11 25 18

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: flutter/ios/Classes/SwiftSentryFlutterPlugin.swift [hunk 1]
```diff
@@ -39,8 +39,7 @@ public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
         infos["integrations"] = integrations
       }
 
-      // TODO get the sdkVersion from SDK
-      infos["package"] = ["version": "6.0.9", "sdk_name": "cocoapods:sentry-cocoa"]
+      infos["package"] = ["version": "\(Options().sdkInfo.version)", "sdk_name": "cocoapods:sentry-cocoa"]
```
Comment (marandaneto) [inline flutter/ios/Classes/SwiftSentryFlutterPlugin.swift]: can we do:

`sentryOptions.sdkInfo.version`?
theres already an instance of it

Comment (denrase) [reply flutter/ios/Classes/SwiftSentryFlutterPlugin.swift]: `sentryOptions` is an optional member of `SwiftSentryFlutterPlugin`. However, it should always have a value when this is called, or is there a code path that would call this before `initNativeSdk`?

Comment (marandaneto) [reply flutter/ios/Classes/SwiftSentryFlutterPlugin.swift]: integrations are executed in order, so `initNativeSdk` is always called before, but I'd definitely check if `sentryOptions` is not null for the sake of safety, if `sentryOptions. sdkInfo` is not available, I'd not set the `infos["package"]` at all.

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**